app/database_etl/test_adjustment_handler/test_adjustment.py

```diff
@@ -103,13 +125,22 @@ def fit_one_pystan_model(self, model_params: Dict) -> Tuple:
 
         samples = fit.extract(pars='prev', permuted=True)['prev']
 
-        summary_df_parsed = summary_df.loc['prev', ['50%', '2.5%', '97.5%']]
+        summary_df_parsed = summary_df.loc['prev', ['mean','50%', '2.5%', '97.5%','n_eff','Rhat']]
         summary_df_parsed['samples'] = samples
         summary_df_parsed['fit'] = fit
-
-        diagnostics = pystan.diagnostics.check_hmc_diagnostics(fit)
-
-        satisfactory_model_found = all(diagnostics.values())
+        
+        
+        # Model Diagnostics using Rhat and N-eff
+        if summary_df_parsed['Rhat'] > 0.95 and summary_df_parsed['Rhat'] < 1.05 or summary_df_parsed['n_eff'] > 1000:
```

Comment:
Do we need both Rhat and n_eff, or either?
Format with parentheses to make order of operations clear

```
(summary_df_parsed['Rhat'] > 0.95 and summary_df_parsed['Rhat'] < 1.05) or (summary_df_parsed['n_eff'] > 1000):
```

Reply:
I prefer to use both. I'll make this an 'and' statement instead of an 'or' statement.